fortium-qbo/app/services/qbo_service.py

```
@@ -236,9 +237,9 @@ def get_company_by_code(self, code: str) -> QboCompany:
         """
         company = self.db.query(QboCompany).filter(QboCompany.code == code).first()
         if not company:
-            raise ValueError(f"QBO company not found: {code}")
+            raise QboNotFound(f"QBO company not found: {code}")
         if company.token_status == "disconnected":
```

Ruff (0.16.1)

[error] Possible hardcoded password assigned to: "token_status"

(S105)